### PR TITLE
HACDOCS-509: Adding info about secrets to the Privacy notice

### PR DESCRIPTION
**Pull request by missmesss**
HACDOCS-509: Updating [the privacy notice](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/support/data-privacy) with info about secrets.

### CODE REVIEW

**Comment by missmesss** [comment]
@jseseCCS Jocelyn, do you mind peer reviewing the changes to the Privacy notice, please?
Also, as you know this topic deeper than I do, maybe you'd suggest adding other bits of info too?

As for the title, I'd prefer a verb-based and user-focused one (e.g. Store sensitive data in secrets) but I tried to keep the title in line with the other 2 titles. WDYT?

Thank you in advance!

Links for your convenience:
Currently published note: https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/support/data-privacy/#data-in-deployment-variables-is-public
Preview of a new note: https://pr-194--boisterous-meerkat-894982.netlify.app/documentation/main/support/data-privacy/#secrets-for-sensitive-data-are-available

**Comment by jseseCCS** [comment]
> @jseseCCS Jocelyn, do you mind peer reviewing the changes to the Privacy notice, please? Also, as you know this topic deeper than I do, maybe you'd suggest adding other bits of info too?
> 
> As for the title, I'd prefer a verb-based and user-focused one (e.g. Store sensitive data in secrets) but I tried to keep the title in line with the other 2 titles. WDYT?
> 
> Thank you in advance!
> 
> Links for your convenience: Currently published note: https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/support/data-privacy/#data-in-deployment-variables-is-public Preview of a new note: https://pr-194--boisterous-meerkat-894982.netlify.app/documentation/main/support/data-privacy/#secrets-for-sensitive-data-are-available

sure thing! reviews shouldn't happen concurrently. peer comes after sme. lmk when Ann Marie's done and then i'll hop in and give this a look-see.

**Comment by missmesss** [comment]
@jseseCCS I'm aware of the instructions, but sometimes flexibility is beneficial too.
In this case my train of thought was: as a writer who dived deeper into the topic, you could suggest what other info could be beneficial to the users, thus acting as a semi-sme, so to say. Because if you suggest something after Ann Marie looks at the PR, we'll need her attention again and I'd prefer to save her time.

**Comment by jseseCCS** [comment]
> @jseseCCS I'm aware of the instructions, but sometimes flexibility is beneficial too. In this case my train of thought was: as a writer who dived deeper into the topic, you could suggest what other info could be beneficial to the users, thus acting as a semi-sme, so to say. Because if you suggest something after Ann Marie looks at the PR, we'll need her attention again and I'd prefer to save her time.

you could make that argument for any content review, but i understand. i try not to add anything in my peer reviews that goes beyond style/grammar, specifically to avoid that scenario. the semi-SME part, though, makes sense. i know @amfred is swamped. (hi, AMF!)

i'll go ahead and take a look, make some comments, then we'll see what AM comes back with. does that sound OK?

File: docs/modules/ROOT/pages/support/data-privacy.adoc
Comment (jseseCCS) [inline docs/modules/ROOT/pages/support/data-privacy.adoc:25]: I'd join this line with the preceding one.

Re the xref, we're supposed to [avoid referencing guides](https://redhat-documentation.github.io/supplementary-style-guide/#cross-references) now.

Change to: To create secrets, see link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_creating-secrets-for-your-builds[Creating secrets for your builds].

**Review by jseseCCS — CHANGES_REQUESTED** [review]

**Comment by missmesss** [comment]
> i try not to add anything in my peer reviews that goes beyond style/grammar
That's great to hear! Thank you. Some writers go far beyond that.

> i'll go ahead and take a look, make some comments, then we'll see what AM comes back with. does that sound OK?
Thanks for being flexible and having a look earlier, appreciate it.

File: docs/modules/ROOT/pages/support/data-privacy.adoc
Comment (missmesss) [reply docs/modules/ROOT/pages/support/data-privacy.adoc:19]: Yes, I believe our goal was to emphasize we've got secrets now. And the updated text is about secrets and not so much about env variables.

I agree, it's not exactly necessary to change the title, on the other hand, it's not like they all must be identical. Technically the predicate is of the same type, to be + adjective.
I first thought to get an action-based title (like Store sensitive data in secrets) but that would be too far from other titles.

File: docs/modules/ROOT/pages/support/data-privacy.adoc
Comment (missmesss) [reply docs/modules/ROOT/pages/support/data-privacy.adoc:21]: Nice, thank you, more concise.

File: docs/modules/ROOT/pages/support/data-privacy.adoc
Comment (missmesss) [reply docs/modules/ROOT/pages/support/data-privacy.adoc:25]: Yeah, a separate line with a link only isn't looking that good, thanks.

But I use the same full-length link, not an xref, I'll keep it then.

**Comment by missmesss** [comment]
@jseseCCS I've pushed the changes you suggested. Please, let me know what you think about the title. Thank you.

File: docs/modules/ROOT/pages/support/data-privacy.adoc
Comment (jseseCCS) [reply docs/modules/ROOT/pages/support/data-privacy.adoc:19]: looks great!

File: docs/modules/ROOT/pages/support/data-privacy.adoc
Comment (amfred) [inline docs/modules/ROOT/pages/support/data-privacy.adoc:23]: This all looks good to me; however, the document that this points to could use an update: 
https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_creating-secrets-for-your-builds

That document makes it sounds like you can ONLY store the Snyk token in the build secret, but you can also store other key-value pairs there.

Comment (amfred) [reply docs/modules/ROOT/pages/support/data-privacy.adoc:23]: One nit - I think you probably want to have `because {ProductName} stores...` instead of `because App Studio stores...`?

**Review by amfred — APPROVED** [review]
Approving just in case I was wrong and you really did want to put `App Studio` there... also I think that's minor.

**Comment by missmesss** [comment]
@amfred Thanks for approving. You're right, it should be {ProductName} indeed.

File: docs/modules/ROOT/pages/support/data-privacy.adoc
Comment (missmesss) [reply docs/modules/ROOT/pages/support/data-privacy.adoc]: > the document that this points to could use an update: https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_creating-secrets-for-your-builds
> 
> That document makes it sounds like you can ONLY store the Snyk token in the build secret, but you can also store other key-value pairs there.

@jc-berger I believe you authored the above mentioned guide. Given you're with the ACM now, will you have capacity to address Ann Marie's feedback?

**Comment by missmesss** [comment]
@jseseCCS Is there anything else we need to discuss or you could provide your lgtm?

**Comment by jseseCCS** [comment]
@missmesss LGTM! could you f/u with Jake, though, about updating that upstream secrets content, then LMK?

**Review by jseseCCS — APPROVED** [review]

File: docs/modules/ROOT/pages/support/data-privacy.adoc
Comment (jseseCCS) [reply docs/modules/ROOT/pages/support/data-privacy.adoc]: @amfred @missmesss I'm comfortable resolving this. Oss will follow up with Jake about updating this content ASAP.

**Comment by missmesss** [comment]
Thanks for approving @jseseCCS  I've already tagged Jake in this PR re Ann Marie's comment (it got hidden in a resolved thread but we all get notified regardless). I'll let you know what we decide ofc.

**Pull request merged**
